src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+      src="images/imac.jpeg"
+      alt="card-image"
+      >
+  
+      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__id">Product code: 195434</p>
+  
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__line"></div>
+          <div class="stars__line"></div>
+          <div class="stars__line"></div>
+          <div class="stars__line"></div>
+          <div class="stars__line"></div>
+        </div>
+        <p class="card__rating-reviews">Reviews: 5</p>
+      </div>
+  
+      <div class="card__price">
+        <p class="card__text-title">Price:</p>
+        <p class="card__text-cost">$2,199</p>
+      </div>
+  
+      <a href="#"
+       class="card__button"
+       data-qa="hover"
+      >
+      BUY
+      </a>
```

Comment:
```suggestion
      <a
        href="#"
       class="card__button"
       data-qa="hover"
      >
        buy
      </a>
```
make uppercase via css